internal/pkg/fetch/fetch.go

```diff
@@ -56,15 +56,11 @@ func Fetch(sbomURL string, opts *options.FetchOptions) error {
 		return fmt.Errorf("error adding document: %w", err)
 	}
 
-	if opts.OutputFile == nil || opts.OutputFile.Name() == "" {
-		return nil
-	}
-
 	// Fetch externally referenced BOMs
-	return fetchExternalReferences(document, backend, opts)
+	return fetchExternalReferences(document, backend, opts, outputFile)
```

Comment:
I think this should remain how it was, referencing `opts.OutputFile` and without the unneeded `outputFile` param.